src/trace-plugin-loader.ts

```diff
@@ -43,15 +130,15 @@ function checkLoadedModules() {
     }
   }
   if (process._preload_modules && process._preload_modules.length > 0) {
-    var first = process._preload_modules[0];
+    const first = process._preload_modules[0];
     if (first !== '@google-cloud/trace-agent') {
       logger_.error('Tracing might not work as ' + first +
             ' was loaded with --require before the trace agent was initialized.');
     }
   }
 }
 
-function checkPatch(patch) {
+function checkPatch(patch: any) {
```

Comment:
Can we avoid `any`?

Reply:
We can avoid it by explicitly setting its type to `{ file?: any, patch?: any, unpatch?: any, intercept?: any }`, or something more complex. I don't think it's worthwhile to set the type more precisely than `any`, given that this function is doing type-checking of some sort on its own.

Reply:
Could we do better than `any` now?